### DISCUSSION TITLE
Plugin-Sitemap not creating sitemap.xml

### DISCUSSION THREAD

**Discussion by soccershoe** [community/community#22035] [Other Feature Feedback, Questions, & Ideas]
<p>Hello.  I’m creating for myself a simple blog using Github Pages.  I’d like to use the ‘Plugin-Sitemap’ feature to automatically create the sitemap.xml file for my site.  But I have so far been unsuccessful at getting the sitemap.xml to be created.  </p>
<p>I’m not running a local build.  I’m pushing only to Pages.  </p>
<p><a href="https://github.com/jekyll/jekyll-sitemap" rel="nofollow noopener">https://github.com/jekyll/jekyll-sitemap</a></p>
<p>I’ve put the following in my _config.yaml:</p>
<pre><code>plugins:
  - jekyll-sitemap
</code></pre>
<p>Do I need to add something to the front matter on each page, like:</p>
<pre><code>sitemap: true
</code></pre>
<p>What am I missing from my config to autogenerate the sitemap.xml?<br>
Thank you.</p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hi <a class="mention" href="/soccershoe">@soccershoe</a> !</p>
<p>Thank you for reaching out!  If you are still having issues with this, can you check to see if you find an XML file loaded when you append “/sitemap.xml” to the end of your GitHub Pages site? <br>
Here is an example of the URL format that I am talking about <a href="https://soccershoe.github.io/PRETENDtestRepo/sitemap.xml" rel="nofollow noopener">https://soccershoe.github.io/PRETENDtestRepo/sitemap.xml</a></p>
<p>The <code>jekyll-sitemap</code> plugin is used to generate a search engine optimized link map for your website, so it won’t show any visual changes on your page when you enable this plugin.<br>
Let me know if that helps.</p>

**Comment by chesun** [comment]
<p>Hi <a class="mention" href="/nyahbhinghiprincess1">@nyahbhinghiPrincess1</a> , wanted to revive this old post as I am running into the same problem. I can see that there is indeed an xml sitemap file when I put the url in my browser, but Google Search Console is saying it fails to fetch the sitemap file from that url. What might be causing this problem? Thank you.</p>

**Comment by Otacon** [comment]
<p>Same issue here. I’ve uploaded the <a href="https://otacon.github.io/practical-kotlin/sitemap.xml" rel="noopener nofollow ugc">sitemap.xml</a> 3 weeks ago.</p>
<p>But Google has not indexed it yet.</p>
<p>I’ve looked on Google and someone said that it might just take some time for Google to index it. Followed lots of weird tutorials, but still, Google is saying “Couldn’t fetch”.</p>
<p>I’m generating the xml with the jekyll plugin. Any luck <a class="mention" href="/chesun">@chesun</a>? Should I just wait?</p>

**Comment by chesun** [comment]
<p>I am also using the jekyll plugin and so far no luck in getting the sitemap to fetch. I manually requested indexing on all my urls and sometimes certain pages would show up on google searches but they are really low in ranking. For whatever reason my website started showing up at the top of searches two days ago all of a sudden, and frankly I have no idea what happened.</p>
